{"version":3,"sources":["../../src/error/AlreadyHasActiveConnectionError.ts"],"names":[],"mappings":";;;;;;;;;;;;AAAA;;GAEG;AACH;IAAqD,mDAAK;IAGtD,yCAAY,cAAsB;QAAlC,YACI,iBAAO,SAIV;QAPD,UAAI,GAAG,iCAAiC,CAAC;QAIrC,KAAI,CAAC,OAAO,GAAG,4CAAyC,cAAc,2CAAuC;YACzG,4DAA4D,CAAC;QACjE,KAAI,CAAC,KAAK,GAAG,IAAI,KAAK,EAAE,CAAC,KAAK,CAAC;;IACnC,CAAC;IAEL,sCAAC;AAAD,CAVA,AAUC,CAVoD,KAAK,GAUzD;AAVY,0EAA+B","file":"AlreadyHasActiveConnectionError.js","sourcesContent":["/**\n * Thrown when consumer tries to recreate connection with the same name, but previous connection was not closed yet.\n */\nexport class AlreadyHasActiveConnectionError extends Error {\n    name = \"AlreadyHasActiveConnectionError\";\n\n    constructor(connectionName: string) {\n        super();\n        this.message = `Cannot create a new connection named \"${connectionName}\", because connection with such name ` +\n            `already exist and it now has an active connection session.`;\n        this.stack = new Error().stack;\n    }\n\n}"],"sourceRoot":".."}